modules/ROOT/pages/manage-versions.adoc

```diff
@@ -0,0 +1,62 @@
+= Manage Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Manage the versions of an asset by viewing version detail, adding a new version, deleting a version, and changing the lifecycle state for a version.
+
+== View Version Detail
+You can view the following information about a version of an asset:
+
+- Name of the owner who published the asset
+- Date the version was published
+- Lifecycle status
+--
+
+To view the details of a version:
+
+. Click *Manage versions* in the expanded asset details pane.
+. Click the three dots menu and choose *Version detail* from the menu.
+-- 
+== Add New Version of an Asset
+
+If you have contributor or admin access to an asset, you can add a new version for an asset. The new version of the asset has the same name, icon, and description as the previous version. Any changes to these properties apply to all versions of the asset.
+
+. Click *Manage versions* in the expanded asset details pane.
+. Click *Add new version*.
+. (Required for APIs) Select a *Method* for the API specification.
+. (Required for APIs) From *File upload* click *Choose file*. For OAS, choose an OAS file with the .yaml or .json file types. For RAML, choose a RAML file with the .raml file type. Zip files are also accepted with the file to upload residing in the zip file’s root directory.
+. Specify the *Main file* if the uploaded file is a zip. Otherwise, Exchange uses the same file name as the file you upload. 
```

Comment:
```suggestion
. Specify a *Main file* value if the uploaded file is a ZIP. Otherwise, Exchange uses the same file name as the name of the file you upload. 
```